packages/core/src/App/Components/Layout/Footer/toggle-fullscreen.jsx

```diff
@@ -55,7 +55,7 @@ const ToggleFullScreen = () => {
                 {is_full_screen ? (
                     <Icon icon='IcFullScreenRestore' className='footer__icon' />
                 ) : (
-                    <Icon icon='IcFullScreen' className='footer__icon' />
+                    <Icon data_testid='dt_icon' icon='IcFullScreen' className='footer__icon' />
```

Comment:
Can't we use a ternary operator here? something like this:
```
<Icon icon={is_full_screen ? 'IcFullScreenRestore' : 'IcFullScreenRestore'} className='footer__icon' />
```
This can remove repeating the `className` and the component itself.
Also, can we not use the icon name itself to test it instead of adding `testid` to one condition?

Reply:
AFAIK we can't get svg icons by text and their name, because of that I used testid for getting it, there are some examples for this but the structure of their svgs were different from us,
If you have any suggestion for getting it without testid please let me know about it, will change my code :smiley:

Reply:
About using ternary operator for the icon, since this is out of the scope of this card I prefer to don't fix it here.

Reply:
> If you have any suggestion for getting it without testid please let me know about it
I think it needs some changes in the icon component and it's out of the scope of this PR. So please ignore it for now.

> About using the ternary operator for the icon, since this is out of the scope of this card I prefer to don't fix it here.
You're right. agreed :+1: